src/OpenTK.GraphicsLibraryFramework/Enums/ContextRobustnessAttribute.cs

```diff
@@ -0,0 +1,16 @@
+namespace OpenToolkit.GraphicsLibraryFramework
+{
+    /// <summary>
+    /// Context robustness attribute.
+    /// </summary>
+    /// <seealso cref="IGLFW.WindowHint(ContextRobustnessAttribute,Robustness)"/>
+    public enum ContextRobustnessAttribute
```

Comment:
We don't need a separate enum/overload for this one single value. It would probably be better to have all the window hints in one enum.

Reply:
but all in one would be bad for type safety. as it has specific parameters for the second parameter of WindowHint